### DISCUSSION TITLE
Clear explanation of Preset (NLP / TLP) and QR-code usage needed (for me;-) )

### DISCUSSION THREAD

**Discussion by VreekFonq** [gopro/labs#1144] [Q&A]
Hi All,
For say a Time-lapse Photo QR code to work, there needs to be a preset on the Go Pro itself regarding Time-lapse Photo.
Is there a strict demand on naming conventions for these presets? Or will the camera get it, when i name the preset Labs-TLP?
Thanks in advance and kind regards, Ramon

**Comment by dnewman-gpsw** [comment]
When you selecting modes in Labs, you are selecting presets. Unfortunately the camera sees photo and video presets as different even though they might both be time lapse or night lapse.  Doesn't matter what you call them.  The issue is it comes pre-configured only with video timelapse presets, no photo presets, you have to manually create those.
